File: mario-more/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{
    // Taking input.
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n <= 0 || n >= 9);

    // Printing mario.
    for (int i = 0; i < n; i++) // For printing new line.
    {
        for (int k = n; k > i + 1; k--) // For printing spaces in between the pyramids.
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++) // For printing left #.
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < i + 1; j++) // For printing right #.
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Taking input from user
while (True):
    Height = get_int("Height: ")
    if (Height > 0 and Height < 9):
        break

for i in range(1, Height+1):
    # Printing spaces
    print(" " * (Height - i), end="")
    # Printing left '#'
    print("#" * i, end="")
    print(" ")
    # Printing right '#'
    print("#" * i, end="")
